openfga/v1/openfga_service.proto

```diff
@@ -390,6 +391,116 @@ service OpenFGAService {
         "}\n"
         "```\n"
         "The caller can then call expand API for the `writer` relationship for the `document:2021-budget`."
```

Comment:
```suggestion
        "The caller can then call expand API for the `writer` relationship for the `document:2021-budget`.\n"
```

otherwise, formatting is incorrect:

<img width="955" alt="image" src="https://github.com/user-attachments/assets/52af8114-54a5-42c6-9512-47b4f3a522b6">